src/component/Header.jsx

```diff
@@ -1,8 +1,58 @@
 import React from "react";
-
+import "../assets/myStyle/header.scss";
 class Header extends React.Component {
     constructor(props) {
         super(props);
+
+        this.state = {
+            activeLink: "Home"
+        }
+    };
+
+    createActiveLink = (event) => {
+        event.preventDefault();
+        event.target.getAttribute("data-name");
+        this.setState({
+            activeLink: event.target.getAttribute("data-name")
+        })
+    };
+
+    render() {
+        return(
+            <div className="header-container">
+                <div className="container">
+                    <header className="header">
+                        <div>
+                            <a href="#" className="logo">Logo</a>
+                        </div>
+                        <nav className="nav">
+                            <ul className="nav-list">
+                                <li>
+                                    <a href="#" data-name="Home"
+                                        onClick={this.createActiveLink}
+                                       className={`nav-item ${this.state.activeLink === "Home"? "is-active": ""}`}>Home
+                                        <span/>
```

Comment:
What is these empty spans are all about?

Reply:
Empty spans are needed for animating links, namely for hover effect.